webviews/src/open-chat/OpenChatApp.tsx

```
@@ -0,0 +1,630 @@
+  modelLabel: string;
+  models: OpenChatOption[];
+  modelId: string;
+  reasoningLabel: string;
+  reasoningLevels: OpenChatOption[];
+  reasoningId: string;
+  onModelSelect: (id: string) => void;
+  onReasoningSelect: (id: string) => void;
+}) {
+  return (
+    <div className="open-chat-menu open-chat-model-menu" role="menu" aria-label={label}>
+      <div className="open-chat-menu-section" aria-label={modelLabel}>
```

Biome (2.5.0)

[error] The ARIA attribute 'aria-label' is not supported by this element.

(lint/a11y/useAriaPropsSupportedByRole)